### ISSUE TITLE
Setting disk SKU

### ISSUE THREAD

**Issue by robbell** [Azure/ansible-aks#14]
Is it possible to choose which SKU disks are provisioned at? I can set size using `node_osdisk_size_gb`, but disks seem to default to `Premium SSD`, at least when using the `Standard_B2s` VM size.

**Comment by darrenjones24** [comment]
I don't believe there would be a way to set the disk SKU using this role; the role uses the azure.azcollection.azure_rm_aks module which does not have that option:
https://docs.ansible.com/ansible/latest/collections/azure/azcollection/azure_rm_aks_module.html

**Comment by robbell** [comment]
Thanks, @darrenjones24 - so this should actually be a request for a change in https://github.com/Azure/azure_modules?
